src/components/view/ContactInfo/contactInfoEventsCard.js

```diff
@@ -0,0 +1,65 @@
+import React, { Component } from 'react';
+import ReactDOM from 'react-dom';
+
+import { Card, CardHeader, CardText } from 'material-ui/Card';
+import { List, ListItem } from 'material-ui/List';
+
+import { getImageForEnv } from 'static/images/index'
+
+class ContactInfoEventsCard extends Component {
+
+    formatUidToDate = uid => {
+        const splitUid = uid.split('');
+        return `${splitUid.slice(4,6).join('')}/${splitUid.slice(7,9).join('')}/${splitUid.slice(0,4).join('')}`
```

Comment:
Can you add a brief comment above this to the effect of: "Converting abc123 to abc 1/2/3"? #recommended

Let's get a 3rd opinion on the readability of this. Typically, I avoid using interpolation (`\``) and instead use concatenation across strings and variables. #recommended

Reply:
mind if I ask why you don't like string interpolation? I agree on the readability, and I'll for sure add comments. wasn't sure if this code would stick around based on how our event uids may or may not change in the future

Reply:
@payne-chris-r  or @camden-hill thoughts?